components/shared/AddressInput.vue

```diff
@@ -77,4 +101,13 @@ const handleInput = (value: string) => {
 
   return props.emptyOnError && error.value ? '' : value
 }
+
+const handleAddressCheck = (isValid: boolean) => {
+  emit('check', isValid)
+  addressCheckError.value = !isValid
```

Comment:
?

Reply:
I've renamed the variable maybe it makes more sense know , let me know if it doesn't